$NetBSD: patch-bindings_python_pyproject.toml,v 1.3 2025/09/15 17:35:53 adam Exp $

Python module version should be obtained from git, but git repository
is not available in PyPI package, so set the version manually.

--- bindings/python/pyproject.toml.orig	2025-03-07 11:43:00.000000000 +0000
+++ bindings/python/pyproject.toml
@@ -4,7 +4,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "unicorn"
-dynamic = ["version"]
+version = "2.1.4"
 requires-python = ">= 2.7, != 3.0.*, != 3.1.*, != 3.2.*, != 3.3.*, != 3.4.*, != 3.5.*, != 3.6.*"
 dependencies = [
     "importlib_resources; python_version < '3.9'"
@@ -42,6 +42,3 @@ test = [
 
 [tool.setuptools.packages.find]
 include = ["unicorn*"]
-
-[tool.setuptools_scm]
-root = '../../'
